core/controllers/acl_decorators.py

```diff
@@ -487,6 +487,44 @@ def test_can_manage_account(self, **kwargs):
     return test_can_manage_account
 
 
+def can_manage_own_profile(handler):
+    """Decorator to check whether user can manage their profile.
```

Comment:
Nit: ``Decorator to check whether a user can manage their profile.``

Reply:
Done.